apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -139,6 +140,9 @@ broadcast(FsmRef, Type, From, Packet) ->
 stop(FsmRef) ->
     ?GEN_FSM:send_event(FsmRef, closed).
 
+store_info(FsmRef, User, Server, Resource, KV) ->
+    FsmRef ! {store_info, User, Server, Resource, KV, self()}.
```

Comment:
and then Elvis will say "don't use macros as module names"... maybe we better leave it as it is.